src/PharoThings-Devices-LCD1602.package/PotLCD1602Device.class/instance/pulseEnable.st

```diff
@@ -0,0 +1,7 @@
+controlling
+pulseEnable
+	(gpioArray at: 2) value: 0.
+	(gpioArray at: 2) value: 1.
+	(gpioArray at: 2) value: 0.	
```

Comment:
the index 2 here is a magic number. But in fact you operate with clock pin. It would be much cleaner if it will be written as:
```Smalltalk
clockPin value: 0; value: 1; value: 0
```
Look comment for #initialize